### ISSUE TITLE
Fail on a server startup because of server.id length limitation

### ISSUE THREAD

**Issue by ilya-chumakov** [frankhommers/Hangfire.PostgreSql#113]
Hi, our Hangfire server broke down completely last week because of `varchar(100)` limit for the `server.id` column in the db. Our devops team did some maintenance stuff and changed host names. As `server.id` depends on the host name, its summary length exceeded 100 chars, leading to an unhandled exception on insert: 

https://github.com/frankhommers/Hangfire.PostgreSql/blob/11152c84e0f12f74b5b3f9366c82fd735d2dad14/src/Hangfire.PostgreSql/PostgreSqlConnection.cs#L392

As a temporary solution, we patched the db manually but I'd like to propose increasing that limit or even altering the column type to `text` in future releases.

Packages:

    <PackageReference Include="HangFire" Version="1.7.3" />
    <PackageReference Include="Hangfire.PostgreSql" Version="1.6.0" />

Exception details:
```
BackgroundExecution      Execution BackgroundServerProcess is still in the Failed state for 00:00:30.0163993 due to an exception, will be retried no more than in 00:00:15
Npgsql.PostgresException (0x80004005): 22001: value too long for type character varying(100)
   at Npgsql.NpgsqlConnector.<>c__DisplayClass161_0.<<ReadMessage>g__ReadMessageLong|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Npgsql.NpgsqlConnector.<>c__DisplayClass161_0.<<ReadMessage>g__ReadMessageLong|0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming)
   at Npgsql.NpgsqlDataReader.NextResult()
   at Npgsql.NpgsqlCommand.ExecuteDbDataReader(CommandBehavior behavior, Boolean async, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteNonQuery(Boolean async, CancellationToken cancellationToken)
   at Npgsql.NpgsqlCommand.ExecuteNonQuery()
   at Dapper.SqlMapper.ExecuteCommand(IDbConnection cnn, CommandDefinition& command, Action`2 paramReader) in C:\projects\dapper\Dapper\SqlMapper.cs:line 2827
   at Dapper.SqlMapper.ExecuteImpl(IDbConnection cnn, CommandDefinition& command) in C:\projects\dapper\Dapper\SqlMapper.cs:line 570
   at Dapper.SqlMapper.Execute(IDbConnection cnn, String sql, Object param, IDbTransaction transaction, Nullable`1 commandTimeout, Nullable`1 commandType) in C:\projects\dapper\Dapper\SqlMapper.cs:line 443
   at Hangfire.PostgreSql.PostgreSqlConnection.AnnounceServer(String serverId, ServerContext context)
   at Hangfire.Server.BackgroundServerProcess.CreateServer(BackgroundServerContext context)
   at Hangfire.Server.BackgroundServerProcess.Execute(Guid executionId, BackgroundExecution execution, CancellationToken stoppingToken, CancellationToken stoppedToken, CancellationToken shutdownToken)
   at Hangfire.Server.BackgroundProcessingServer.RunServer(Guid executionId, Object state)
   at Hangfire.Processing.BackgroundExecution.Run(Action`2 callback, Object state)
```

**Comment by frankhommers** [comment]
We might as well change all VARCHAR(n) fields to TEXT for this since it's their both similar performance wise. Can you make a new Install*.sql file in a PR that does this?

**Comment by vytautask** [comment]
Should be solved with 1.6.1 version.
